VulnerableApplication/Models/Hello.cs

```diff
@@ -11,5 +11,9 @@ public void Say()
         {
             Console.WriteLine("Hello!");
         }
+    public string Hi() {
+    return "this is against the rules.";
```

Comment:
[CODE_SMELL] Replace this string literal with a string retrieved through an instance of the 'ResourceManager' class.